setup.py

```diff
@@ -5,19 +5,17 @@
 from typing import Dict, List
 
 import setuptools
-from setuptools import setup
+from setuptools import find_packages, setup
 
 _PACKAGE_NAME = 'mosaicml-examples'
-_PACKAGE_DIR = 'mosaicml_examples'
-_EXAMPLE_SUBDIRS = ('cifar', 'resnet', 'deeplab', 'bert', 'llm')
-
-_project_root = os.path.dirname(os.path.realpath(__file__))
-
-print('_project_root:', _project_root)
+_EXAMPLE_SUBDIRS = ['cifar', 'resnet', 'deeplab', 'bert', 'llm']
+_PACKAGES = find_packages(exclude=['*tests*', '*scripts*'])
```

Comment:
Is the goal still to just have `examples` be the one package? I think this will make any subdir with an __init__.py a package, which might be unintended behavior down the road.